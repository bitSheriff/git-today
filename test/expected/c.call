../target/debug/git-today repos/c
