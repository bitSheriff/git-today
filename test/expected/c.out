╭─────────┬──────────────╮
│ Author  ┆ # of Commits │
╞═════════╪══════════════╡
│ Author1 ┆       6      │
╰─────────┴──────────────╯
╭─────────────┬──────────────╮
│ Issue Type  ┆ # of Commits │
╞═════════════╪══════════════╡
│ 🐛 Bugs     ┆       2      │
├╌╌╌╌╌╌╌╌╌╌╌╌╌┼╌╌╌╌╌╌╌╌╌╌╌╌╌╌┤
│ 🚀 Features ┆       1      │
├╌╌╌╌╌╌╌╌╌╌╌╌╌┼╌╌╌╌╌╌╌╌╌╌╌╌╌╌┤
│ 📝 Docs     ┆       2      │
├╌╌╌╌╌╌╌╌╌╌╌╌╌┼╌╌╌╌╌╌╌╌╌╌╌╌╌╌┤
│ 🧬 Merges   ┆       1      │
╰─────────────┴──────────────╯
